canonical-web-and-design/jaas-dashboard
# Clear the model data on log out and stop polling

## Done

- When logging out:
  - Stop polling the model statuses and model lists.
  - Clear the stored model data.
- Various refactoring to prepare for the changes required to fix #47 

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Perform the following steps with the redux store open
- After logging in check that it loads your models as expected.
- After 30s from completing it should restart the polling.
- If you log out the polling should stop and it should not restart or fetch the model list.
- Try logging out during active polling, and after all polling has stopped (during the 30s delay).

Note: You will have to refresh the page after logging out else you won't be able to log back in again (see #47)

## Details

Fixes #218
